src/sessions/session-manager.ts

```diff
@@ -293,22 +346,31 @@ export class SessionManager {
     const session = this.getSession(params);
 
     const sessionHandler = this.getSessionHandler({ jingleSession: session });
+
     return sessionHandler.endSession(session, params.reason);
   }
 
   async setVideoMute (params: ISessionMuteRequest): Promise<void> {
-    const session = this.getSession({ sessionId: params.sessionId });
+    const session = this.getSession({ conversationId: params.conversationId });
 
     const handler = this.getSessionHandler({ sessionType: session.sessionType });
     await handler.setVideoMute(session, params);
   }
 
   async setAudioMute (params: ISessionMuteRequest): Promise<void> {
-    const session = this.getSession({ sessionId: params.sessionId });
+    const session = this.getSession({ conversationId: params.conversationId });
 
     const handler = this.getSessionHandler({ sessionType: session.sessionType });
     await handler.setAudioMute(session, params);
   }
+
+  async setConversationHeld (params: IConversationHeldRequest): Promise<void> {
```

Comment:
this could be problematic. What if softphone and screenshare are happening at the same time?

Reply:
What is supposed to happen then? This was just something I copied over from web-dir because it really helped out my testing. If there needs to be extra logic involved, I'll create a ticket and pull this out.

Reply:
The problem is determining which session you actually want. if we have both sessions active at the same time, it's going to return the first one for the conversation. If it pull the screenshare session and we call `setConversationHeld` on that handler, it's going to throw and error (also it's not going to hold the conversation).

Again this isn't necessarily a problem with what exists currently, but I'm thinking down the road. We could also just say "let's do it this way for now and we'll cross the collision bridge when we get there."

Or we could also just get this pr merged then handle the conversationId stuff in a separate pr, but before we release v7.